ext/com_dotnet/com_extension.c

```diff
@@ -237,8 +237,8 @@ static PHP_INI_MH(OnTypeLibFileUpdate)
 		modifier = php_strtok_r(NULL, "#", &strtok_buf);
 		if (modifier != NULL) {
 			if (!strcmp(modifier, "cis") || !strcmp(modifier, "case_insensitive")) {
-				php_error_docref("com.configuration", E_DEPRECATED, "Declaration of case-insensitive constants is deprecated");
-				mode &= ~CONST_CS;
+				php_error_docref("com.configuration", E_WARNING, "Declaration of case-insensitive constants is no longer supported");
+				return FAILURE;
```

Comment:
I wonder if it would be better to just continue on here, possibly with a rephrasing of the warning to say that the case insensitive annotation will be ignored.

Aborting here is a bit odd because it might end up with a part of the typelibs being loaded and a part not.